bidtastic/app/assets/javascripts/items.js

```diff
@@ -1,2 +1,29 @@
 // Place all the behaviors and hooks related to the matching controller here.
 // All this logic will automatically be available in application.js.
+
+$(document).ready(function() {
+
+  // Favoriting
+  $('.TBD').on('click', '.fav-heart', function(event) {
+    let homeId = $(this).data("itemid")
```

Comment:
I think this is copy/paste from another project so make sure your variable names are appropriate for *this* project.